backend/editor/entries.py

```diff
@@ -206,7 +225,13 @@ def update_node_children(entry, new_children_ids):
     existing_ids = [record['child.id'] for record in get_current_transaction().run(query, ids=list(added_children))]
     to_create = added_children - set(existing_ids)
 
-    for child in to_create:
+    #Normalising new children node ID
+    normalised_added_children = set()
+    for child_node in to_create:
+        child_language_code, new_child_name = child_node.split(":",1)
+        normalised_added_children.add(normalizing(new_child_name,child_language_code))

```

Comment:
You have to put back language code !
```suggestion
        normalised_added_children.add(
            f"{child_language_code}:{normalizing(new_child_name, child_language_code)}")

```

Reply:
but see below because I propose you to remove this part.